RecoJets/JetProducers/python/PileupJetID_cfi.py

```diff
@@ -34,6 +33,11 @@
 #     residualsTxt     = cms.FileInPath("RecoJets/JetProducers/data/download.url") # must be an existing file
 )
 
+# set algos for re-miniAOD campaign
+run2_miniAOD_80XLegacy.toModify(pileupJetId, algos = _chsalgos_81x)
+run2_miniAOD_94XFall17.toModify(pileupJetId, algos = _chsalgos_94x)
+(run2_miniAOD_UL & run2_jme_2017).toModify(pileupJetId, algos = _chsalgos_106X_UL17)
+
```

Comment:
for the EOY18 now it takes the UL `_chsalgos_106X_UL18`
should be instead set the `_chsalgos_102x` ?

if yes, then the following should work
```
_stdalgos = _chsalgos_102x
(run2_miniAOD_UL ).toModify(pileupJetId, algos = _chsalgos_106X_UL18)
(run2_miniAOD_UL & run2_jme_2017).toModify(pileupJetId, algos = _chsalgos_106X_UL17)
```

Reply:
> `_stdalgos = _chsalgos_102x`

I'm not sure I understand this suggestion: as written, it will apply to all new PU ID, while the goal is to use `_chsalgos_106X_UL18` for everything made from scratch and for Run3.

Reply:
ok, I see why `_stdalgos = _chsalgos_106X_UL18`
but then we give up on the EOY18 and _chsalgos_102x ?

Reply:
IIRC, we do not have a re-miniAOD workflow using EOY18 as inputs.
or is it incorrect?

Reply:
> IIRC, we do not have a re-miniAOD workflow using EOY18 as inputs.

[to elaborate] running miniAOD on `EOY18` would imply a separate configuration with a specific modifier (`run2_miniAOD_*`). Such modifier does not exist. Hence, there is nothing to modify here to use `_chsalgos_102x`

Reply:
> [to elaborate] running miniAOD on `EOY18` would imply a separate configuration with a specific modifier (`run2_miniAOD_*`). Such modifier does not exist. Hence, there is nothing to modify here to use `_chsalgos_102x`

Indeed, for the mini EOY18 is lost here and for everything else. 
Note once we have the veryUL we will loose the UL18 because we do not have the (run2_miniAOD_UL & run2_jme_2018).